javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java

```diff
@@ -28,23 +30,23 @@
     @Autowired
     private JaversProperties javersProperties;
 
-    @Autowired
-    private ConnectionProvider connectionProvider;
-
     @Bean
-    public Javers javers() {
-        JaversRepository javersRepository = SqlRepositoryBuilder.sqlRepository()
+    public Javers javers(ConnectionProvider connectionProvider) {
+        JaversSqlRepository sqlRepository = SqlRepositoryBuilder
+                .sqlRepository()
```

Comment:
done, tam jest coś takiego:

`Database.DEFAULT`

i to jest chyba jakoś w runtime rozkminiane, u nas dałem po prostu H2